traffic_ops/ort/atstccfg/toreq/toreq.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/lib/go-util"
-	toclient "github.com/apache/trafficcontrol/traffic_ops/client"
+	toclient "github.com/apache/trafficcontrol/traffic_ops/v2-client"
```

Comment:
I don't think this is going to have the desired effect on `atstccfg` due to the way it currently vendors the TO-Go client. I'm pretty sure this will make it use the non-vendored v2 client. I think we need to follow the process of re-vendoring the latest version of the TO-Go client in atstccfg. Maybe that should be done in a separate follow-up PR, and just leave ORT as-is for now?

Reply:
Yeah, this is vendored. It's not necessary to change this, unless API 1 is being removed from Traffic Ops. If so, the `vendor/` dir needs updating along with this import.

The latest client is in https://github.com/apache/trafficcontrol/blob/master/traffic_ops/ort/atstccfg/toreqnew/toreqnew.go

But it's not necessary to update that either unless there's a new feature ORT needs.

Reply:
ah, I see